display.js

```diff
@@ -0,0 +1,105 @@
+    featuredImage: 'assets/images/tonic2.svg',
+    technologies: ['HTML', 'CSS', 'ReactJs'],
+    linkToLiveVersion: '#',
+    linkToSource: '#',
+  },
+  {
+    id: '03',
+    name: 'Tonia 3',
+    role: 'Developer',
+    platform: 'Web',
+    year: '2022',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. ',
+    descriptionlong:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent ',
+    featuredImage: 'assets/images/poststories.svg',
+    technologies: ['HTML', 'CSS', 'ReactJs'],
+    linkToLiveVersion: '#',
+    linkToSource: '#',
+  },
+  {
+    id: '04',
+    name: 'Tonia 4',
+    role: 'Developer',
+    platform: 'Web',
+    year: '2022',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. ',
+    descriptionlong:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent ',
+    featuredImage: 'assets/images/poststories2.svg',
+    technologies: ['HTML', 'CSS', 'ReactJs'],
+    linkToLiveVersion: '#',
+    linkToSource: '#',
+  },
+
+];
+
+const modalContainer = document.createElement('div');
+modalContainer.className = 'popup';
+const modalOpen = (id) => {
+  const closeBtn = '&times;';
+  modalContainer.innerHTML = `
+<div class="work">
+      <div class="card">
+       <button type="button" onClick="closeModal()">${closeBtn}</button>
+        <img class="card-img" src="${portfolioData[id].featuredImage}" alt="poststories">
+        <div class="content">
+          <div class="tonic">
+            <h3>${portfolioData[id].name}</h3>
+            <ul>
+              <li><a href="#">Canopy <img src="assets/images/Counter.svg" alt=""></a></li>
+              <li><a href="#"> Back End Dev <img src="assets/images/Counter.svg" alt=""></a></li>
+              <li><a href="#"> 2015</a></li>
+            </ul>
+          </div>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="lang">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">Javascript</a></li>
+          </ul>
+          <button class="btn btn-outline btn-hover mt-15 btn-pressed">See Project</button>
+        </div>
+      </div>
+    </div>
+`;
+  document.body.appendChild(modalContainer);
+};
```

Comment:
- [ ]  Your modal window does not look like the Figma. Please take a look at the comparison below and fix the discrepancies before resubmission.

**Figma design**
![figma1](https://github.com/Izebiafe/afimiaportfolio/assets/99656687/6d86b645-e8d4-408b-adec-73e4a9189cf2)

**Your implementation**
![scrn](https://github.com/Izebiafe/afimiaportfolio/assets/99656687/5c1b78db-c31e-4cb0-9ddd-fbe457f61dc3)

**Required action**
- [ ] Kindly position the close button as in the Figma 😄 
- [ ] The close button does not work when clicked, please fix 🔧 this
- [ ] The image should be above the text, not beside 👁️ 
- [ ] The languages & button are besides the  text not below ⚠️

Reply:
- [ ] The pop that is displayed when the project below is clicked is not the right popup. Please fix this so that the correct project details are displayed 👍🏾 
![scrn](https://github.com/Izebiafe/afimiaportfolio/assets/99656687/9327b1b8-6ca6-4f13-9e32-5bcfb44361d6)

** This is the popup  that is displayed**
![crn](https://github.com/Izebiafe/afimiaportfolio/assets/99656687/985a55a6-61c6-436a-b5dc-c28a8093b2b1)